src/Illuminate/Notifications/resources/views/email.blade.php

```diff
@@ -18,12 +18,12 @@
 
 {{-- Action Button --}}
 @isset($actionText)
-<?php
+@php
     $color = match ($level) {
         'success', 'error' => $level,
         default => 'primary',
     };
-?>
+@endphp
```

Comment:
You could also do
```suggestion
@php($color = in_array($test, ['success', 'error']) ? $level : 'primary')
```

Reply:
Yes, it is possible to refactor this code as you wrote it, but I think it was written this way for extensibility and customization by developers. (`match` is used)

Reply:
In that case, the logic shouldn't be in the template at all in the first place but in a method to be called in the template